packages/suite/src/views/backup/index.tsx

```diff
@@ -12,38 +12,57 @@ import { FAILED_BACKUP_URL } from '@suite-constants/urls';
 import type { InjectedModalApplicationProps } from '@suite-types';
 import type { BackupStatus } from '@backup-actions/backupActions';
 
-const Row = styled.div`
-    display: flex;
-    flex-direction: row;
+const StyledButton = styled(Button)`
+    width: 224px;
+`;
+
+const StyledP = styled(P)`
+    color: ${props => props.theme.TYPE_LIGHT_GREY};
 `;
 
-const Col = styled.div`
-    display: flex;
-    flex-direction: column;
+const StyledImage = styled(Image)`
+    margin-bottom: 24px;
+    align-self: center;
 `;
 
-const Buttons = styled(Row)`
-    justify-content: center;
+const VerticalCenter = styled.div`
     margin-top: auto;
+    margin-bottom: auto;
 `;
 
-const StyledButton = styled(Button)`
-    min-width: 224px;
-    margin-bottom: 16px;
+const NestedModal = styled(Modal)`
+    height: 620px;
```

Comment:
It was because one of the modals was always nested and the other one was never nested, so their default css was different. But I've checked that and you're right that currently it's not needed anymore.
https://github.com/trezor/trezor-suite/pull/5132/commits/0db62a8f7b1e818d070fdcc5c425ac2938a18bbc